src/redux/store.test.ts

```diff
@@ -25,6 +27,16 @@ const resetStateOnInvalidStoredAccount = jest.spyOn(
 
 const loggerErrorSpy = jest.spyOn(Logger, 'error')
 
+const getNonApiReducers = <R = Omit<RootState, ApiReducersKeys>>(state: RootState): R => {
+  const apiReducersKeys: string[] = ['transactionFeedV2Api'] satisfies ApiReducersKeys[]
```

Comment:
Could we do this instead? to avoid having to update multiple files when we update them.
```suggestion
  const apiReducersKeys: string[] = Object.keys(apiReducersList)
```